core/integration/secret_test.go

```diff
@@ -41,7 +41,8 @@ func TestSecretEnvFromFile(t *testing.T) {
 			"secret": secretID,
 		}})
 	require.NoError(t, err)
-	require.Contains(t, envRes.Container.From.WithSecretVariable.Exec.Stdout, "SECRET=some-content\n")
+	// secret should be scrubbed
+	require.Contains(t, envRes.Container.From.WithSecretVariable.Exec.Stdout, "SECRET=***\n")
```

Comment:
Seems like these tests should be tweaked now that they don't have such a strong assertion.

Maybe the command could run its own assertion instead?

* `sh -c 'test "$SECRET" = "some-content"'` for env test
* `sh -c 'test "$(cat /sekret)" = "some-content"'` for file mount tests

Reply:
Yeah, I wanted to improve that as well, but after our exchange, I thought it was impossible that way. But this idea to test in the container is great. I'll fix that!

Reply:
done